docs/ansible/roles/debops.borgbackup/inventory/group_vars/debops_service_borgbackup.yml

```diff
@@ -0,0 +1,8 @@
+---
+# vim: foldmarker=[[[,]]]:foldmethod=marker
+
+sudo__group_sudoers:
+
+  - name: 'backup-mgt-borgmatic'
+    raw: |
+      backup-mgt ALL = (ALL) NOPASSWD: /usr/local/bin/borgmatic --create --stats
```

Comment:
I think the path should be dynamic, ether if it is install through package manager or though pip

Reply:
Read `SUDOERS(5)` ;)

As you said already, installing via pip is only done up until buster. After that, it will by installed from Debian. If it where not a workaround, `borgbackup__bin_path` could be used.